gtfs_go_loader.py

```diff
@@ -127,6 +137,20 @@ def canceled(self):
         self.loadingAborted.emit()
         self.close()
 
+    def raised_error(self, error):
```

Comment:
In my opinion, this method should be named as meaning it's function or timing when are triggered.

example:  ``show_error_message()`` or ``on_raised_error()``

Reply:
Thank you, i fixed it